pkgs/applications/graphics/vimiv-qt/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python39Packages
+, qt5
```

Comment:
```suggestion
, qt5
, installShellFiles
```

required for installManPage

Reply:
Done with the last commit, sorry forgot to resolve the conversation.